interfaces/repo.go

```diff
@@ -973,23 +923,26 @@ func (r *Repository) SnapSpecification(securitySystem SecuritySystem, appSet *Sn
 	return spec, nil
 }
 
-// AddSnap adds plugs and slots declared by the given snap to the repository.
+// AddAppSet adds plugs and slots declared by the given snap to the repository.
```

Comment:
this doc comment probably need a bit more updates, as the AppSet is not strictly just about the snap anymore